lab-bundle-spec.md

```diff
@@ -271,13 +269,90 @@ No additional attributes
   id: primary_domain
 ```
 
-##### Future Resource Types
+##### AWS Account (aws_account)
+
+attribute   | required | type       | notes
+----------- | -------- | ---------- | ----------------------------------------
+account_restrictions.allow_dedicated_instances  |          | boolean | Default false.
+account_restrictions.allow_spot_instances       |          | boolean | Default false.
+account_restrictions.allow_subnet_deletion      |          | boolean | Default false.
+account_restrictions.allow_vpc_deletion         |          | boolean | Default false.
+account_restrictions.allowed_ondemand_instances |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for Ondemand usage. Default none.
+account_restrictions.allowed_rds_instances      |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for RDS usage. Default none.
+startup_script.type              |          | string  | The type of startup script. Only `cloud_formation` is supported.
+startup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+startup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+cleanup_script.type              |          | string  | The type of cleanup script. Only `cloud_formation` is supported.
+cleanup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+cleanup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+user_policy                      |          | path    | Relative path to a [JSON policy](https://awspolicygen.s3.amazonaws.com/policygen.html) document.
+
+```yml
+- type: aws_account
+  id: the_account
+  variant: sts
+  startup_script:
+    type: cloud_formation
+    path: ./lab.template
+    custom_properties:
+      - key: userNameWindows
+        value: student
+  user_policy: ./student.policy
+  account_restrictions:
```

Comment:
Sorry for the drive-by comment, but I was talking to the AWS CE folks and they were hoping to be able to specify a maximum number of managed compute (Lightsail, SageMaker, EBS) instances. Should that be added here?

Reply:
When/if we add the functionality, that's something we can add here.

Reply:
So, broadly speaking, we are free to add fields to this spec when we want?

Reply:
Broadly speaking, yes.

Reply:
As long as it's backwards compatible (which additive changes usually are)